gui/src/app/view/receive.rs

```diff
@@ -1,34 +1,44 @@
 use iced::{
     widget::qr_code::{self, QRCode},
-    Alignment,
+    Alignment, Length,
 };
 
 use liana::miniscript::bitcoin;
 
 use liana_ui::{
-    component::{card, text::*},
+    component::{button, card, text::*},
     icon, theme,
     widget::*,
 };
 
 use super::message::Message;
 
 pub fn receive<'a>(address: &'a bitcoin::Address, qr: &'a qr_code::State) -> Element<'a, Message> {
-    card::simple(
-        Column::new()
-            .push(QRCode::new(qr).cell_size(10))
-            .push(
-                Row::new()
-                    .push(text(address.to_string()).small())
-                    .push(
-                        Button::new(icon::clipboard_icon())
-                            .on_press(Message::Clipboard(address.to_string()))
-                            .style(theme::Button::TransparentBorder),
-                    )
-                    .align_items(Alignment::Center),
-            )
-            .align_items(Alignment::Center)
-            .spacing(20),
-    )
-    .into()
+    Column::new()
+        .push(card::simple(
+            Column::new()
+                .push(QRCode::new(qr).cell_size(10))
+                .push(
+                    Row::new()
+                        .push(text(address.to_string()).small())
+                        .push(
+                            Button::new(icon::clipboard_icon())
+                                .on_press(Message::Clipboard(address.to_string()))
+                                .style(theme::Button::TransparentBorder),
+                        )
+                        .align_items(Alignment::Center),
+                )
+                .align_items(Alignment::Center)
+                .spacing(20),
+        ))
+        .push(
+            Column::new().push(
+                button::primary(None, "Next")
```

Comment:
Maybe
```suggestion
                button::primary(None, "Generate new")
```

What do you all think?

Reply:
That looks good to me. Will make this change.